advanced/config.yaml

```diff
@@ -0,0 +1,90 @@
+components:
+  schemas:
+    AdvancedMetrics.Config:
+      description: |-
+        The container for advanced metrics configuration.
+      type: object
+      required: [choice]
+      properties:
+        choice:
+          type: string
+          enum: [events, convergence, latency]
+        events:
+          $ref: '#/components/schemas/AdvancedMetrics.Config.Events'
+        convergence:
+          $ref: '#/components/schemas/AdvancedMetrics.Config.Convergence'
+        latency:
+          $ref: '#/components/schemas/AdvancedMetrics.Config.Latency'
+
+    AdvancedMetrics.Config.Events:
+      type: object
+      properties:
+        enable:
+          description: |-
+            True to enable raw event metrics. 
+            See the #/components/schemas/Event.Occurence/properties/event
+            property for a list of all possible types of events.
+          type: boolean
+          default: false
+        rx_rate_threshold:
+          description: |-
+            An event will be generated when the rx rate passes
+            above or below this value.
+          type: number
+          maximum: 100
+          format: float
+          default: 95
+
+    AdvancedMetrics.Config.Convergence:
+      description: |-
+        The optional container for convergence configuration options.
+      type: object
+      properties:
+        enable:
```

Comment:
Should enabling convergence automatically enable events as well (and allow fetching event metrics) ?